Permissions.js

```diff
@@ -47,7 +47,7 @@ class Permissions {
     _requestPermissionAndroid(perm) {
         return new Promise((resolve, reject) => {
             PermissionsAndroid.request(perm).then(
-                granted => resolve(granted === PermissionsAndroid.RESULTS.GRANTED),
+                granted => resolve(granted === true || granted === PermissionsAndroid.RESULTS.GRANTED),
```

Comment:
this workaround seems temporary, do you mind leaving here a FIXME that it's to be removed eventually ?

Reply:
or probably git history will show this commit

Reply:
It's on the git commit message.